src/components/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <div>
+    <nav>
+      <h1>Bookstore CMS</h1>
+      <ul>
+        <li><Link to="/" style={{ textDecoration: 'none' }}>BOOKS</Link></li>
+        <li><Link to="/" style={{ textDecoration: 'none' }}>CATEGORIES</Link></li>
```

Comment:
- [ ] The same problem is here for the link. Please fix this issue.
  -  Change the second link with this code `<li><Link to="/categories" style={{ textDecoration: 'none' }}>CATEGORIES</Link></li>`